docs/rules/comma-dangle.md

```diff
@@ -69,7 +69,8 @@ The default for each option is `"never"` unless otherwise specified.
 * `imports` is for import declarations of ES Modules. (e.g. `import {a,} from "foo";`)
 * `exports` is for export declarations of ES Modules. (e.g. `export {a,};`)
 * `functions` is for function declarations and function calls. (e.g. `(function(a,){ })(b,);`)<br>
-  `functions` is set to `"ignore"` by default for consistency with the string option.
+  `functions` is set to `"ignore"` by default for consistency with the string option.<br>
```

Comment:
Can these two lines be in a sublist instead of forcing line breaks with a <br> tag?